app_usb_aud_xk_evk_xu316/src/extensions/audiohw.xc

```diff
@@ -15,45 +19,6 @@ on tile[1]: out port p_codec_reset  = PORT_CODEC_RST_N;
 // CODEC Reset
 #define CODEC_RELEASE_RESET      (0x8) // Release codec from
 
-// App PLL setup
-#define APP_PLL_CTL_BYPASS       (0)   // 0 = no bypass, 1 = bypass.
-#define APP_PLL_CTL_INPUT_SEL    (0)   // 0 = XTAL, 1 = sysPLL
-#define APP_PLL_CTL_ENABLE       (1)   // 0 = disabled, 1 = enabled.
-
-// 24MHz in, 24.576MHz out, integer mode
-// Found exact solution:   IN  24000000.0, OUT  24576000.0, VCO 2457600000.0, RD  5, FD  512, OD 10, FOD  10
-#define APP_PLL_CTL_OD_48        (4)   // Output divider = (OD+1)
-#define APP_PLL_CTL_F_48         (511) // FB divider = (F+1)/2
-#define APP_PLL_CTL_R_48         (4)   // Ref divider = (R+1)
-
-#define APP_PLL_CTL_48           ((APP_PLL_CTL_BYPASS << 29) | (APP_PLL_CTL_INPUT_SEL << 28) | (APP_PLL_CTL_ENABLE << 27) |\
-                                    (APP_PLL_CTL_OD_48 << 23) | (APP_PLL_CTL_F_48 << 8) | APP_PLL_CTL_R_48)
-
-// Fractional divide is M/N
-#define APP_PLL_FRAC_EN_48             (0)   // 0 = disabled
```

Comment:
I note we lose this tidy ups in lib_sw_pll (see https://github.com/xmos/lib_sw_pll/issues/65)